src/Compilers/CSharp/Test/Semantic/Semantics/NullCheckedParameterTests.cs

```diff
@@ -799,5 +799,61 @@ class Program
                     //     static void Main(string[] args!) { }
                     Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "args").WithArguments("System.ArgumentNullException", ".ctor").WithLocation(4, 31));
         }
+
+        [Fact]
+        public void TestNullCheckedMethodParameterWithWrongLanguageVersion()
+        {
+            var source =
+@"
+class Program
+{
+    void M(string x!) { }
+}";
+            var comp = CreateCompilation(source, parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp6));
+            comp.VerifyDiagnostics(
+                    // (4,20): error CS8059: Feature '!' is not available in C# 6. Please use language version CSharp8 or greater.
```

Comment:
Please add a string resource for `IDS_ParameterNullChecking` in CSharpResources.resx, perhaps with a value like "parameter nullchecking"

Reply:
I should have mentioned this in my earlier feedback, sorry :P

Reply:
already done, was waiting on tests to pass